tests/update_testapps_dependencies.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Update dapr dependencies for all e2e test apps
+cd ./apps
+for i in `ls`
+do
+   if test -f "$i/go.mod"
+   then
+      cd $i > /dev/null
```

Comment:
I would use `pushd` and `popd` here.

Reply:
I modified this a bit. No need to either use push/pop or cd-